db/drivers/mysql/db.c

```diff
@@ -54,7 +54,9 @@ int db__driver_open_database(dbHandle *handle)
 
         connection = mysql_init(NULL);
         res = mysql_real_connect(connection, host, user, password,
```

Comment:
```suggestion
        res =
            mysql_real_connect(connection, host, user, password, connpar.dbname,
```
![image](https://github.com/OSGeo/grass/assets/27212526/bfc3b1a3-d20b-479c-9a00-6f254553ca96)

```
        res =
            mysql_real_connect(connection, host, user, password, connpar.dbname,
                               port != NULL ? atoi(port) : 0, NULL, 0);
```